### PR TITLE
Updated Xamarin.Essentials: Info About PresentationSourceBounds

### PR DESCRIPTION
**Pull request by dimonovdd**
Added information about using `PresentationSourceBounds` property on `Launcher`.

information about `PresentationSourceBounds` property has been moved to a new file and is now included in the documentation for `Share` and `Launcher`

Linked [Issue](https://github.com/xamarin/Essentials/issues/1611) and [Pull Request](https://github.com/xamarin/Essentials/pull/1637)

### CODE REVIEW

**Comment by PRMerger10** [comment]
@dimonovdd : Thanks for your contribution! The author(s) have been notified to review your proposed change. @jamont

**Comment by PRMerger14** [comment]
@dimonovdd : Thanks for your contribution! The author(s) have been notified to review your proposed change. @jamont

**Comment by davidbritch** [comment]
@dimonovdd Give me a nudge when this is ready to be merged.

**Comment by dimonovdd** [comment]
@davidbritch ok

**Comment by dimonovdd** [comment]
@davidbritch @mattleibow @jamesmontemagno 
 [1.6.1-preview2](https://github.com/xamarin/Essentials/releases/tag/1.6.1-preview2) has already been released
I think we can publish this.

**Comment by dimonovdd** [comment]
@davidbritch, Did you see it?

**Review by mattleibow — APPROVED** [review]

**Comment by davidbritch** [comment]
Just to clarify, is this feature available in a stable release, or a preview release?

**Comment by dimonovdd** [comment]
yes this is already in the stable release 1.6.1

File: docs/essentials/launcher.md [hunk 1]
```diff
@@ -92,6 +92,12 @@ await Launcher.OpenAsync(new OpenFileRequest
 });
 ```
 
+## Presentation Location When Opening Files
+
+![Pre-release API](~/media/shared/preview.png)
```
Comment (davidbritch) [inline docs/essentials/launcher.md]: Double checking that the preview image is required?

Comment (dimonovdd) [reply docs/essentials/launcher.md]: I think it's not required

Comment (dimonovdd) [reply docs/essentials/launcher.md]: I'm busy right now and can't delete this

**Comment by PRMerger7** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change. @jamont

**Comment by PRMerger7** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change. @jamont

**Review by davidbritch — APPROVED** [review]
Thanks @dimonovdd

**Pull request merged**